icons/receipt.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M4 2v20l2-1 2 1 2-1 2 1 2-1 2 1 2-1 2 1V2l-2 1-2-1-2 1-2-1-2 1-2-1-2 1-2-1Z" />
```

Comment:
```suggestion
  <path d="M4 2v20l2-1 2 1 2-1 2 1 2-1 2 1 2-1 2 1V2l-2 1-2-1-2 1-2-1-2 1-2-1-2 1-2-1" />
```

There is also this https://github.com/svg/svgo/pull/1751

Reply:
In this instance a [`closepath` command](https://www.w3.org/TR/SVG/paths.html#PathDataClosePathCommand) is preferable, I think, since we don't want any potential for two line caps in a closed path.

Reply:
You can remove the zero witdth close path command there but keeping it is also 100% fine.